lib/drivers/abbottFreeStyle.js

```diff
@@ -373,8 +373,9 @@ module.exports = function (config) {
       // once we've sent the command, start listening for a response
       // but if we don't get one before the end of the timeout, give up
       // max timeout is 1000 * 32 bytes / 19200 baud / 10 bits/char == about 16 sec
```

Comment:
Yes, I'd like to keep the info on what a max timeout will be, even if we only set it to 3s.